packages/sdk/src/peer_manager/peer_manager.ts

```diff
@@ -198,13 +204,14 @@ export class PeerManager {
 
   private async onConnected(event: CustomEvent<IdentifyResult>): Promise<void> {
     const result = event.detail;
-    const isFilterPeer = result.protocols.includes(
-      this.matchProtocolToCodec(Protocols.Filter)
-    );
-
-    if (isFilterPeer) {
+    if (
+      result.protocols.includes(this.matchProtocolToCodec(Protocols.Filter))
```

Comment:
nit: out of scope, but may be worth refactoring this to work with all codecs, protocols, and their corresponding event. perhaps making `dispatchStorePeerConnect` work like `dispatchPeerConnect<Protocols.Store>`